contents/docs/self-host/deploy/digital-ocean.md

```diff
@@ -10,183 +10,153 @@ showTitle: true
-
-    **⚠️ Note: Do not use our placeholder key! Read more about the importance of this key [here](/docs/self-host/configure/securing-posthog).**
-
-    First, run: `openssl rand -hex 32`. This will generate a new key for you. You'll need this in the next step.
-
-    Then, open the `docker-compose.yml` file with the command: `nano docker-compose.yml`
 
-    Lastly, substitute `"<randomly generated secret key>"` for the key you got from the key generation command.
+### 2. Install the chart
 
-    This means the `SECRET_KEY: "<randomly generated secret key>"` line will end up looking something like this (with your key, of course):
+To install the chart with the release name `posthog` in `posthog` namespace, run the following:
 
-    ```
-    SECRET_KEY: "cd8a182315defa70d995452d9258908ef502da512f52c20eeaa7951d0bb96e75"
-    ```
-1. Then, to run PostHog, do:
-```bash
-docker-compose up -d
+```console
+helm repo add posthog https://posthog.github.io/charts-clickhouse/
+helm repo update
+helm install -f values.yaml --timeout 20m --create-namespace --namespace posthog posthog posthog/posthog
 ```
-1. You're good to go! PostHog should be accessible on the domain you set up or the IP of your instance.
-
-<blockquote class='warning-note'>
-
-**Important:** If you do not have a TLS/SSL certificate set up for your domain/IP, accessing the address of your PostHog instance _will not work_. To get around this, you need to edit the `docker-compose.yml` file manually and add the environment variables   `DISABLE_SECURE_SSL_REDIRECT: 'true'` and `SECURE_COOKIES: 'false'` under `services > web > environment`. This is a manual process because PostHog should not be run without a certificate (i.e. over HTTP). 
-
-Doing this and restarting the service will allow you to access PostHog over HTTP, but might require configuring browser settings to allow HTTP traffic depending on what browser you use. 
-
-</blockquote>
-
-<br />
 
-## Running PostHog Behind a Proxy or Load Balancer
+### 3. Lookup external IP
 
-If you're running PostHog behind a proxy or load balancer, you need to set the `IS_BEHIND_PROXY` environment variable to `True`. Depending on your setup, you might also need to set the `ALLOWED_HOSTS` [environment variable](/docs/self-host/configure/environment-variables).
-
-
-For more information, visit our [dedicated page for running PostHog behind a proxy](/docs/self-host/configure/running-behind-proxy).
-
-## Important Points
-
-#### ⚠️ Never, Ever, Run PostHog Without TLS/SSL
-PostHog needs to run on HTTPS because if it doesn't:
- 
- a) It will fail<br />
- b) Your user data will be at risk
-
-#### Check Your Firewall if You Cannot Connect to a Port
-
-If you are unable to connect to a certain port, this might be due to the firewall settings for your droplet. Generally, this is a matter of running:
-
-```bash
-sudo ufw allow <PORT> && sudo ufw reload
-```
-To check that the changes were applied, run: 
-```bash
-sudo ufw status
+```console
```

Comment:
Should this be `bash` ?

Reply:
Seemed like `console` is recommended https://stackoverflow.com/questions/20303826/highlight-bash-shell-code-in-markdown-files